helm/prometheus-meta-operator/templates/configmap.yaml

```diff
@@ -28,8 +28,8 @@ data:
         {{- range .Values.Installation.V1.Monitoring.Prometheus.Bastions }}
         - {{ . }}
         {{- end }}
-        # remoteWrite:
-        #   url: https://prometheus-us-central1.grafana.net/api/prom/push
+        remoteWrite:
+          url: https://prometheus-us-central1.grafana.net/api/prom/push
```

Comment:
I remember this was disabled because we need to swap it off in _g8s-prometheus_ at the same time as enabling it here to avoid duplicating the metrics (https://github.com/giantswarm/prometheus-meta-operator/commit/8def43ce1d3fcced14797fdb4dae40fd46241184). Just double-checking, are we ready to do that? Is there an associated PR for _g8s-prometheus_?

Reply:
We do have the old one ready yes. Once this gets merged, i will release both

Reply:
Just for reference, it's giantswarm/g8s-prometheus#1130.